dev/io.openliberty.microprofile.reactive.messaging.internal_fat/bnd.bnd

```diff
@@ -27,19 +27,22 @@ fat.test.container.images: confluentinc/cp-kafka:7.1.1
 tested.features:\
   concurrent-3.0,\
   cdi-4.0,\
-  servlet-6.0
+  servlet-6.0,\
+  mpconfig-3.1,\
+  mpmetrics-4.0,\
+  mpmetrics-5.1
 
 -buildpath: \
-    io.openliberty.microprofile.reactive.messaging.internal_fat.common.jakarta;version=latest,\
-    io.openliberty.org.eclipse.microprofile.reactive.messaging.3.0;version=latest,\
-    io.openliberty.org.eclipse.microprofile.reactive.streams.operators.3.0;version=latest,\
-    com.ibm.websphere.org.reactivestreams.reactive-streams.1.0;version=latest,\
-    io.openliberty.org.eclipse.microprofile.config.3.0;version=latest,\
-    io.openliberty.jakarta.cdi.3.0;version=latest,\
-    io.openliberty.jakarta.servlet.5.0;version=latest,\
-    org.apache.kafka:kafka-clients;version=3.5.1,\
-    io.openliberty.org.testcontainers;version=latest,\
-    com.ibm.ws.componenttest.2.0;version=latest
+	io.openliberty.microprofile.reactive.messaging.internal_fat.common.jakarta;version=latest,\
+	io.openliberty.org.eclipse.microprofile.reactive.messaging.3.0;version=latest,\
+	io.openliberty.org.eclipse.microprofile.reactive.streams.operators.3.0;version=latest,\
+	com.ibm.websphere.org.reactivestreams.reactive-streams.1.0;version=latest,\
+	io.openliberty.org.eclipse.microprofile.config.3.0;version=latest,\
+	io.openliberty.jakarta.cdi.3.0;version=latest,\
+	io.openliberty.jakarta.servlet.5.0;version=latest,\
+	org.apache.kafka:kafka-clients;version='3.5.1',\
```

Comment:
not sure where the `'` has come from, but should be put bkac

Reply:
I've noticed that if you change stuff via the editor, it will add single quotes around the version like this. It doesn't look like there are any real changes in this part of the file, so ideally the change should be removed so as not to clutter the diff.